README.md

```diff
@@ -65,6 +65,41 @@ Want to install another different tool for a try? No problem. Want to remove or
 | CD/GitOps              | ArgoCD App (manages the Application CRD of ArgoCD) | ArgoCD Application Creation |
 | Monitoring             | Prometheus/Grafana                                 |                             |
 
+## Quick Install
+
+### Binary (Cross-platform)
+
+Download the appropriate dtm version for your platform from [DevStream Releases](https://github.com/merico-dev/stream/releases).
+
+Once downloaded, the binary can be run from anywhere. You don’t need to install it into a global location.
```

Comment:
the binary can be run from anywhere -> you can run the binary from anywhere

Avoid passive voice when possible.